tests/support/cluster.rs

```diff
@@ -61,7 +62,36 @@ impl RedisCluster {
         let status = dbg!(cmd).status().unwrap();
         assert!(status.success());
 
-        RedisCluster { servers, folders }
+        let cluster = RedisCluster { servers, folders };
+        if replicas > 0 {
+            cluster.wait_for_replicas(replicas);
```

Comment:
If we use replicas, we need to wait for the replica information to be passed on to all nodes.